exporter/elasticsearchexporter/integrationtest/datareceiver.go

```diff
@@ -27,16 +27,18 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/testbed"
 )
 
+var emptyLogs plog.Logs
+
 type esDataReceiver struct {
 	testbed.DataReceiverBase
 	receiver receiver.Logs
-	endpoint string
+	Endpoint string
```

Comment:
nit: Any specific reason it needs to be exported? The bench test file is in the same package.

Reply:
I exported it believing it would be important to know about the endpoint even if we want to use it outside the package.

Reply:
I'd be in favor of not exporting this symbol just yet - small API surface, YAGNI, and all that. It would be easy enough to do it later if/when the need arises.